x-pack/plugins/cases/server/client/configure/client.test.ts

```diff
@@ -724,69 +695,186 @@ describe('client', () => {
               casesClientInternal
             )
           ).rejects.toThrow(
-            'Failed to get patch configure in route: Error: Invalid custom field keys: custom_field_key_2'
+            'Failed to get patch configure in route: Error: The following custom fields have the wrong type in the request: "text label"'
           );
         });
 
-        it('throws when template has customField with invalid type in the request', async () => {
+        it('adds new custom field to template when configuration custom fields have new custom field', async () => {
```

Comment:
Could you please add an `update` test where we change the `defaultValue` of an **existing** `customField`(existing in the configuration and in the template, we just update the default) to guarantee that its value does not change in the template?

Reply:
Added in [bfc856f](https://github.com/elastic/kibana/pull/187591/commits/bfc856f3763ba514291153e11d58d9e2714ceeaf)